### ISSUE TITLE
MathJax Support?

### ISSUE THREAD

**Issue by grself** [jbub/ghostwriter#61]
I'm attempting to code mathematics formulas on a blog page but my site is rendering those formulas out as a text string rather than a mathjax formula. For example, I am using this formula:

\hat{y}=\beta_1x+\beta_0

It gets rendered as a text string. I've attempted to use various types of math delimiters, like $...$ and \(...\) but that did not help. I added the mathjax script tag to the header partial file but that did not help. I even attempted a couple of different "mathjax = true" lines in the config.toml but that did not help. 

Is there any way to render a math formula in a blog post? 

The page with the bad math is located at: https://gself.com/2018/10/16/simple-regres/. A couple of paragraphs below the correlogram is this paragraph:

The general regression formula is \(\hat{y}=\beta_1x+\beta_0\) where the output (\(\hat{y}\) is determined by two parameters of the model (\(\beta\) and the input (\(x\). As an example, if \(\beta_1\) is 2 and \(\beta_0\) is 1 then an input (\(x\) of 1 would yield an output (\(\hat{y}\) of 3.

Thanks for any help you can offer.

**Comment by grself** [comment]
I discovered that a block-level math formula seems to work properly, so \[ Y = A + B \] will be formatted as a math formula in the middle of an otherwise empty line, just as it should. However, inline formulas, like \( Y = A + B \), do not work.

**Comment by grself** [comment]
Problem solved. I thought the symptoms were odd since block math formulas worked. After a bit of experimentation, I found that I needed to delete the space I had after the inline delimiter for MathJax to work.

Thus: \\( \\pi r^2 \\) does not work (space after the "\\(" delimiter)

But: \\(\\pi r^2\\) does work (no space after the "\\(" delimiter)

At this point, I do not have the MathJax script link in "extra-in-head.html" but the delimiters have no space around them and it seems to work properly.
